test/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+
+include(FetchContent)
+
+FetchContent_Declare(
+  googletest
+  GIT_REPOSITORY https://github.com/google/googletest/
+  GIT_TAG "v1.16.0"
+)
```

Would be nice if we could please use 
```suggestion 
FIND_PACKAGE_ARGS NAMES googletest GTest GoogleTest
)
````
So that if the dep is available from another source it wont fetch it from the internet.

Thank you!

---

added on new commit